.github/workflows/ci.yml

```diff
@@ -13,6 +13,9 @@ jobs:
   build:
     uses: xoofx/.github/.github/workflows/dotnet.yml@main
     with:
-      dotnet-version: '6.0 8.0'
```

Comment:
Passing `dotnet-version: '6.0 8.0 9.0'` to the setup-dotnet action ends up building the latest LTS + the first item in the list.
With `6.0 8.0` that just happens to match the intended result.